{"name": "输入错误的用户名", "status": "failed", "statusDetails": {"message": "pytest_assume.plugin.FailedAssumption: \n1 Failed Assumptions:\n\ntestcase\\test_login.py:83: AssumptionFailure\n>>\tpytest.assume(flag == False)\nAssertionError: assert False", "trace": "tp = <class 'pytest_assume.plugin.FailedAssumption'>, value = None, tb = None\n\n    def reraise(tp, value, tb=None):\n        try:\n            if value is None:\n                value = tp()\n            if value.__traceback__ is not tb:\n>               raise value.with_traceback(tb)\nE               pytest_assume.plugin.FailedAssumption: \nE               1 Failed Assumptions:\nE               \nE               testcase\\test_login.py:83: AssumptionFailure\nE               >>\tpytest.assume(flag == False)\nE               AssertionError: assert False\n\nD:\\Python\\lib\\site-packages\\six.py:718: FailedAssumption"}, "steps": [{"name": "预期与结果", "status": "passed", "attachments": [{"name": "预期", "source": "7d19a9e1-58ec-4025-95c2-a71b5a406b92-attachment.json", "type": "application/json"}, {"name": "结果", "source": "0b0b7640-cf4d-4253-86ee-e7cc221d3d34-attachment.json", "type": "application/json"}], "start": 1650508611698, "stop": 1650508611699}, {"name": "响应失败原因", "status": "passed", "attachments": [{"name": "数据", "source": "7f2807df-bec8-4c55-a98d-8db1359008b6-attachment.txt", "type": "text/plain"}], "start": 1650508611699, "stop": 1650508611699}, {"name": "断言结果", "status": "passed", "start": 1650508611699, "stop": 1650508611940}], "attachments": [{"name": "接口路径", "source": "a124cf63-883b-4a31-8888-67bce981b1fb-attachment.attach"}, {"name": "请求参数", "source": "30a82502-afd9-4316-8723-cebe733f2cf6-attachment.attach"}, {"name": "预期结果", "source": "ccd1e937-1131-4cc1-8d16-4812326b29be-attachment.attach"}, {"name": "响应状态", "source": "4301cb81-a7f8-4beb-9a96-31426da24166-attachment.attach"}], "parameters": [{"name": "case", "value": "{'case_id': 1, 'case_title': '输入错误的用户名', 'url': 'http://127.0.0.1:6666/business/regist_form', 'case_data': '{\"username\":\"admin\",\"password\":\"admin01\",\"re_password\":\"admin01\",\"phone\":\"15736884464\",\"sex\":\"女\",\"birthday\":\"1995-04-03\",\"qq\":\"12345@qq.com\",\"email\":\"19999@163,com\"}', 'res': '{\"code\": 100, \"msg\": \"用户名必须以字母开头，由字母和数字组成 6-8位的字符串\"}', 'app-name': '登录'}"}], "start": 1650508611676, "stop": 1650508611676, "uuid": "6f861920-b527-4a37-9928-5a8b41b45f80", "historyId": "36f2f3d80fe1d5ee04c6e3c3e14bcb9c", "testCaseId": "f0a3740a932ee8262933e5b94534ef30", "fullName": "testcase.test_login.TestLogin#test_login01", "labels": [{"name": "feature", "value": "master"}, {"name": "story", "value": "{\"code\": 100, \"msg\": \"用户名必须以字母开头，由字母和数字组成 6-8位的字符串\"}"}, {"name": "epic", "value": "输入错误的用户名"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Mikey"}, {"name": "thread", "value": "6000-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_login"}]}